tests/test_did_meta_plugins.py

```diff
@@ -118,7 +118,17 @@ def test_add_did_meta(self, mock_scope, root_account):
         meta_value = 'my_value_%s' % generate_uuid()
         add_did(scope=mock_scope, name=did_name, did_type='DATASET', account=root_account)
         set_metadata(scope=mock_scope, name=did_name, key=meta_key, value=meta_value)
-        assert get_metadata(scope=mock_scope, name=did_name, plugin='JSON')[meta_key] == meta_value
+
+        meta = get_metadata(scope=mock_scope, name=did_name, plugin='JSON')
+        assert meta[meta_key] == meta_value
+        dids = [{"scope": mock_scope, "name": did_name}]
```

Comment:
It might be worth testing this with a list of multiple DIDs, as that would be the use case for `get_metadata_bulk` over `get_metadata`